src/account.ts

```diff
@@ -50,9 +56,12 @@ export const toChecksumAddress = function(address: string): string {
 
 /**
  * Checks if the address is a valid checksummed address.
+ *
+ * If a chainId is provided, the address is compared to an EIP-1191 checksum. Otherwise, it will be
```

Comment:
Can you also re-generate the docs?

Reply:
I thought they were regenerated when a new release is published

Reply:
Hmm, we still don't have a clear policy here. Until now I had also some tendency to the "regenerate the docs once the code changes" variant. On a second thought the docs are primarily for the users using the releases, so probably the suggestion from @alcuadrado to generate on release makes more sense. @s1na what do you think?

Reply:
Just like you I had no clear approach in mind, but I think regenerating them on release does make sense. Let's go with that.

Reply:
TBH I'm not sure why I had that in mind, but I think one of its advantages is that PRs are smaller.